### ISSUE TITLE
Cell coordinates not read with NVDA+tab when "cell coordinates" unchecked in document formatting.

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#8581]
### Steps to reproduce:

With "Cell coordinates" checked in NVDA's document formatting dialog, move to any cell in an Excel worksheet and press NVDA+tab.  NVDA reads "Cell F4" (or whatever cell you are in).

### Actual behavior:

Open NVDA's document formatting dialog, uncheck "cell coordinates" and save.

On any cell in Excel, press NVDA+tab.  NVDA reads "Cell ".

### Expected behavior:

Expected behaviour would be that NVDA+tab should read the cell coordinates in both cases.  No other information is provided by NVDA+tab and it is actually most useful when cell coordinates are turned off in document formatting for a user to double check where they are (since with the setting off it doesn't announce as they move).

### System configuration:

Windows 10 Fast Insider 17723
Excel 2016 / Office 365 version 1807, build 10325.20082

#### NVDA Installed/portable/running from source:
NVDA 2018.2.1 (Portable)
NVDA Alpha 15736,ef9be392 (Installed)

**Comment by DrSooom** [comment]
Same result with NVDA 2018.1 and Microsoft Excel 2010 on Win7x64.

**Comment by Qchristensen** [comment]
Just confirming the issue still exists in Windows 10 version 18362.1 NVDA 2019.2 and Office 365 version 16.0.11929.20296

**Comment by CyrilleB79** [comment]
> ### Expected behavior:
> Expected behaviour would be that NVDA+tab should read the cell coordinates in both cases. No other information is provided by NVDA+tab and it is actually most useful when cell coordinates are turned off in document formatting for a user to double check where they are (since with the setting off it doesn't announce as they move).

Note that NVDA+NumpadDot (desktop layout) allows to check the current cell coordinates even with the option disabled in doc formatting settings.
Do you still think that cell coordinates info should be added to NVDA+Tab report that usually does not report position informations in other applications?
I agree in the case of empty cell that NVDA+Tab reports few information; however when the cell is not empty, maybe the user wants only the cell content and not the additional position information.

**Comment by Qchristensen** [comment]
You are right, NVDA+numpad delete (NVDA+delete in laptop keyboard layout) does read the cell coordinates in both cases.  Ok, maybe a better comment would be that if it is going to report "cell", it should also report the coordinates, otherwise drop speaking "cell".  I haven't got a set opinion either way on which one of those things should happen though.

**Comment by CyrilleB79** [comment]
I do not agree with dropping "cell". Indeed, NVDA+Tab seems to always report the object type and the object text or content. Dropping "cell" would mean you do not report anymore the object type for Excel cells. This would not be coherent with other NVDA+Tab usages.
